internal/rpc/core/routes.go

```diff
@@ -28,6 +28,7 @@ func (env *Environment) GetRoutes() RoutesMap {
 		"block_results":        rpc.NewRPCFunc(env.BlockResults, "height", true),
 		"commit":               rpc.NewRPCFunc(env.Commit, "height", true),
 		"check_tx":             rpc.NewRPCFunc(env.CheckTx, "tx", true),
+		"remove_tx":            rpc.NewRPCFunc(env.RemoveTx, "txkey,removeFromCache", false),
```

Comment:
Is `removeFromCache` an argument. I thought we'd removed it?

Reply:
yes, sorry missed this case and the compiler wasn't much help :)